code/game/machinery/atmoalter/canister.dm

```diff
@@ -277,6 +299,10 @@ update_flag
 /obj/machinery/portable_atmospherics/canister/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = NANOUI_FOCUS)
 	if (src.destroyed)
 		return
+	if (sealed == TRUE)
```

Comment:
Again, looks like something that should be checked in Topic when you actually attempt doing something.
The way it is now you wouldn't be able to see contents / pressure etc because gasket is sealed.

Reply:
Actually lad, when you examine a canister while standing next to it, you get a description of its contents' pressure and temperature. That's why I did it this way - the nanoUI window doesn't need to be brought up at all, you can just shift examine the can.

Reply:
Ended up doing this your way anyway jerkkkk